app/repositories/system.py

```diff
@@ -101,3 +103,25 @@ async def _handle_gather_hardware_info():
 async def register_hardware_info_gatherer():
     loop = asyncio.get_event_loop()
     loop.create_task(_handle_gather_hardware_info())
+
+
+async def shutdown(reboot: bool):
+    params = ""
+    if(reboot):
+        params = "reboot"
+    script = f"{SHELL_SCRIPT_PATH}config.scripts/blitz.shutdown.sh"
```

Comment:
I get this when I run this on my Blitz: 
`Mar 21 19:34:59 sudo[381301]: bash: /home/adminconfig.scripts/blitz.shutdown.sh: No such file or directory`

In Python there is the function os.path.join to join different paths together:
https://www.geeksforgeeks.org/python-os-path-join-method/

Nice way to avoid pitfalls like this :D